public/js/selfoss-events.js

```diff
@@ -174,7 +174,7 @@ selfoss.events = {
                         selfoss.filter.sourcesNav = true;
                     }
                 } else if (selfoss.events.subsection != 'all') {
-                    selfoss.ui.showError('Invalid subsection: '
+                    selfoss.ui.showError(F + ': '
```

Comment:
What is this?

Reply:
Hotkey fail I didn't notice until reviewing it in Github. 😅